### ISSUE TITLE
docs - broken link to joyent

### ISSUE THREAD

**Issue by RichardWright** [joyent/node-verror#89]
Hi

The link to http://www.joyent.com/developers/node/design/errors in the readme is broken. I've tried to find the new location but can't seem to find it. Any ideas on where it might have moved to?

**Comment by RichardWright** [comment]
I guess it's this? 

https://www.tritondatacenter.com/node-js/production/design/errors

**Comment by bahamat** [comment]
Yeah, sorry about the broken link. Joyent has put in a lot of redirects for us, but we don't have 100% coverage, and as you can guess there are a *lot* of links in our repos that need updating. I'll see if I can get Joyent to add a redirect for that.
